subprojects/distributions-dependencies/build.gradle.kts

```diff
@@ -15,7 +15,7 @@ description = "Provides a platform dependency to align all distribution versions
 
 val antVersion = "1.10.13"
 val archunitVersion = "1.0.0-rc1"
-val asmVersion = "9.4"
```

Comment:
❓  Is there any risk to just bump it to latest, e.g. 9.6?

Reply:
I'd be cautious with that - let's check the master's version (9.5), and if it's good, we can bump it higher.